### PR TITLE
Update the-loader

### PR DESCRIPTION
**Pull request by ACraft4**
Some code to do the task

### CODE REVIEW

**Comment by mvchn** [comment]
I can't see any button here, 
Manual tests are not passed

File: index.html [hunk 1]
```diff
@@ -5,56 +5,22 @@
     <meta charset="UTF-8">
     <title>-X-</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
```
Comment (mvchn) [inline index.html:8]: External styles are ok, but I don't see any reason to do this.

File: mtrx.html [hunk 2]
```diff
@@ -9,17 +9,22 @@
...
     <link rel="icon" type="image/png" sizes="192x192"  href="/favicon/android-icon-192x192.png">
     <link rel="icon" type="image/png" sizes="96x96" href="/favicon/favicon-96x96.png">
-    <style>
-        body {
-            margin: 0;
-            height: 100vh;
-            width: 100vw;
-        }
-    </style>
+    <script src="https://cdn.jsdelivr.net/npm/pixi.js@5.2.4/dist/pixi.min.js"></script>
```
Comment (mvchn) [inline mtrx.html:12]: External javascript is sometimes okay, but we should be able to work without external network connection

**Review by mvchn — CHANGES_REQUESTED** [review]

File: mtrx.html (hunk 2 above)
Comment (ACraft4) [reply mtrx.html:12]: This was a mistake, I delete this in next commit

**Pull request closed without merging**